packages/frontend/amp/components/Standfirst.tsx

```diff
@@ -22,47 +23,65 @@ const listStyles = (pillar: Pillar) => css`
         height: 12px;
         width: 12px;
         margin-right: 8px;
-        background-color: ${palette.neutral[86]};
+        background-color: ${pillarPalette[pillar].neutral.border};
         margin-left: -20px;
     }
 `;
 
-const standfirstCss = pillarMap(
-    pillar => css`
-        ${headline(2)};
-        font-weight: 100;
-        color: ${palette.neutral[7]};
-        margin-bottom: 12px;
-        ${listStyles(pillar)};
-        p {
-            margin-bottom: 8px;
-            font-weight: 200;
-        }
-        strong {
-            font-weight: 700;
-        }
-    `,
-);
+const standfirstCss = (pillar: Pillar) => css`
+p,
+    ${headline(2)};
+    font-weight: 100;
+    color: ${palette.neutral[7]};
+    margin-bottom: 12px;
+    margin-bottom: 12px;
+    ${listStyle(pillar)};
+    p {
+        margin-bottom: 8px;
+        font-weight: 200;
+    }
+    strong {
+        font-weight: 700;
+    }
+`;
 
 const standfirstLinks = pillarMap(
     pillar =>
         css`
             a {
                 color: ${pillarPalette[pillar].dark};
                 text-decoration: none;
-                border-bottom: 1px solid ${palette.neutral[86]};
+                border-bottom: 1px solid ${pillarPalette[pillar].neutral.border};
+            }
+            a:hover {
+                border-bottom: 1px solid ${pillarPalette[pillar].dark};
```

Comment:
What are your thoughts on removing the `composeLabsCSS` stuff completely?

I'm not sure this is much clearer, it does have an advantage on being more contextual on what styles are applied and will be compiled out on pages it doesn't match on (I wanted to test my assumption locally, hence the weird prettierfied formatting on that opening `...)

```suggestion
                border-bottom: 1px solid ${pillarPalette[pillar].dark};
            }

             p,
            li {
                ${pillar === 'labs' &&
                    `
                font-weight: 700;
                ${textSans(8)}
                }`}
```

Reply:
Actually, better:
```js
${pillar === 'labs' &&
                    `
                p,
                li {
                    font-weight: 700;
                    ${textSans(8)}
                }`}
```

Reply:
My first version of this PR used conditional rendering within the CSS like this, but the worry is that there are a lot of hard coded 'labs' variables around the code base.  The idea behind `composeLabsCSS` was to use emotion composition to move the conditional logic into the javascript classNames so that it's clear looking at the component JSX that this component is impacted by labs. The utility function also has the benefit of hardcoding the variable in one place. 

However, making changes to the PasteUp Pallette greatly reduced the need for conditional rendering (just covers font changes now) so perhaps not as big an issue to revert back - I'm ok either way. 

I personally think we could do a lot more in Pasteup palettes to include all the neutral colors so that everything in a component is determined by Pillar, not a single hardcoded neutral palette color anywhere. I also think we could create a typography "Palette" that functions the same way, making any kind of conditional CSS logic out of the components and into the Pasteup, but that felt out of scope of this PR.

Reply:
> I also think we could create a typography "Palette" that functions the same way, making any kind of conditional CSS logic out of the components and into the Pasteup, but that felt out of scope of this PR.

Yup, this sounds great actually, lets think on that idea a bit!